package.json

```diff
@@ -35,6 +35,7 @@
     "acorn-stage3": "^2.0.0",
     "acorn-walk": "^6.2.0",
     "cheerio": "^1.0.0-rc.3",
+    "codecov": "^3.6.5",
```

Comment:
Why is this a dependency instead of devDependency?

Reply:
That's indeed an issue, it should not.

Reply:
Can you fix this maybe ?

Reply:
https://github.com/Polyconseil/easygettext/pull/78